src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -133,43 +133,46 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _UpdateOverrideSystem();
 
         // Get the correct base to apply automation properties to
-        DependencyObject base{ nullptr };
+        std::vector<DependencyObject> base;
+        base.reserve(2);

```

Comment:
You can use `til::small_vector<DependencyObject, 2> base;` here and drop the reserve. Or `til::some` (although we might want to remove til::some in favor of the more generic `til::small_vector`).

Reply:
`DependencyObject` doesn't have a default ctor, so neither work :(